indico/modules/events/abstracts/templates/abstract.html

```diff
@@ -32,7 +32,8 @@
     {% endcall %}
 
     <div class="i-timeline">
-        {% if abstract.can_judge(session.user, check_state=true) and abstract.event.tracks %}
+        {% if abstract.event.cfa.allow_convener_track_change and abstract.can_convene(session.user, check_state=True)
+            and abstract.event.tracks %}
```

Comment:
i'd just keep it in one line (we don't have a hard line length limit in templates) or add parentheses..